src/redux/note-details/calculate-line-start-indexes.test.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { calculateLineStartIndexes } from './calculate-line-start-indexes'
+
+describe('calculateLineStartIndexes', () => {
+  it('works with an empty list', () => {
+    expect(calculateLineStartIndexes([])).toEqual([])
+  })
+  it('works with an non empty list', () => {
```

Comment:
Maybe add a test case that empty lines work as well?
(Though they should work, it saves us of future mistakes)